docs/README.md

```diff
@@ -1,29 +1,35 @@
 # Build documentation with MkDocs
 
 OceanBase documentation is built with [MkDocs](https://www.mkdocs.org/). You can check [`mkdocs.yml`](mkdocs.yml) for more information.
+Please install MkDocs according to [the installation documents of MkDocs](https://www.mkdocs.org/user-guide/installation/)
 
 ## Requirements
 
 Before installing dependencies, please make sure you have installed a recent version of Python 3 and pip.
 
 Then you can run the following command in your terminal at current directory:
 
-    $ pip install -r requirements.txt
-
+```
+$ pip install -r requirements.txt
+$ pip install mkdocs-material
```

Comment:
I think it's better to add `mkdocs-material` into requirements.txt

Reply:
ok